internal/contour/adapter.go

```diff
@@ -76,7 +77,7 @@ func (d *DAGAdapter) OnDelete(obj interface{}) {
 
 func (d *DAGAdapter) setIngressRouteStatus(statuses dag.IngressrouteStatus) {
 	for _, s := range statuses.GetStatuses() {
-		fmt.Println(fmt.Sprintf("DAGVer: %d IR: %s Namespace: %s Status: %s Msg: %s", statuses.GetVersion(), s.GetIngressRouteName(), s.GetIngressRouteNamespace(), s.GetStatus(), s.GetMsg()))
+		d.IngressRouteStatus.SetStatus(s.GetMsg(), s.GetStatus(), s.GetObject())
```

Comment:
Do we need to handle this error?

Reply:
I wasn't sure if we wanted to pipe through a logger but I did that and have this error handled now.